tests/forms_tests/tests/test_forms.py

```diff
@@ -3953,3 +3953,10 @@ class CommentForm(Form):
             '<div class="error">This field is required.</div></div>'
             '<p>Comment: <input type="text" name="comment" required></p>',
         )
+
+    def test_label_debug(self):
+        class Name(Form):
+            first_name = CharField()
+            template_name_label = 'forms_tests/debug.html'
```

Comment:
Can you include the `forms_tests/debug.html` template?

Reply:
🤦‍♂️

Done.

It will need a rename, as I was originally making a custom debug node but it's not really that anymore. (If indeed we need it)

Reply:
Maybe:
```suggestion
            template_name_label = 'forms_tests/cyclic_context_boundfield_render.html'
```